paasta_tools/cli/cmds/validate.py

```diff
@@ -156,10 +156,27 @@ def add_subparser(subparsers):
     validate_parser.set_defaults(command=paasta_validate)
 
 
+def check_service_path(service_path):
+    """Check that the specified path exists and has yaml files
+
+    :param service_path: Path to directory that should contain yaml files
+    """
+    if not service_path or not os.path.isdir(service_path):
+        print failure("%s is not a directory" % service_path,
+                      "http://paasta.readthedocs.org/en/latest/yelpsoa_configs.html")
+        return 1
```

Comment:
That is fine. The reason I went for the numeric approach is to allow us to easily extend this command to have multipole failure return values that we can handle in different ways. But I am fine changing it to the cleaner True/False for now.